<!DOCTYPE html>
<html lang="en-US">
	<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
		<!-- General meta -->
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<link rel="icon" href="favicon.png" type="image/x-icon">

		

		
			<!-- Begin Jekyll SEO tag v2.3.0 -->
<title>Projects | Bioinformatics Technology Lab</title>
<meta property="og:title" content="Projects">
<meta property="og:locale" content="en_US">
<meta name="description" content="Bioinformatics Technology Lab at Canada’s Michael Smith Genome Sciences Centre. Primarily interested in de novo genome assembly. More description.">
<meta property="og:description" content="Bioinformatics Technology Lab at Canada’s Michael Smith Genome Sciences Centre. Primarily interested in de novo genome assembly. More description.">
<link rel="canonical" href="http://localhost:4000/projects.html">
<meta property="og:url" content="http://localhost:4000/projects.html">
<meta property="og:site_name" content="Bioinformatics Technology Lab">
<script type="application/ld+json">
{"name":null,"description":"Bioinformatics Technology Lab at Canada’s Michael Smith Genome Sciences Centre. Primarily interested in de novo genome assembly. More description.","author":null,"@type":"WebPage","url":"http://localhost:4000/projects.html","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/assets/logo.png"}},"image":null,"headline":"Projects","dateModified":null,"datePublished":null,"sameAs":null,"mainEntityOfPage":null,"@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->

		

    <link rel="stylesheet" href="/assets/styles.css">

		
	</head>
	<body class="layout-page  projects">

		<header class="header">
  <div class="container">

    <a class="logo" href="/">
  <img src="/assets/logo.png" alt="Bioinformatics Technology Lab logo">
</a>


    
<nav class="nav  nav--header">
  <ul class="list  list--nav">
    

      

      <li class="item  item--nav">
        <a href="/">Home</a>
      </li>
    

      

      <li class="item  item--nav">
        <a href="/members">Members</a>
      </li>
    

      

      <li class="item  item--nav">
        <a href="/publications">Publications</a>
      </li>
    

      

      <li class="item  item--nav">
        <a href="/projects">Projects</a>
      </li>
    

      

      <li class="item  item--nav">
        <a href="/software">Software</a>
      </li>
    

      

      <li class="item  item--nav">
        <a href="/news">News</a>
      </li>
    
  </ul>
</nav>


<template id="buttontoggle">
  <button class="button  button--nav" aria-label="Nav toggle">
    <svg width="16" height="16" class="icon  icon--nav" role="img"><use xlink:href="#nav"></use></svg>

  </button>
</template>

<script type="text/javascript">

const nav = document.querySelector('.nav')
const buttonTemplate = document.querySelector('#buttontoggle')
const button = document.importNode(buttonTemplate.content, true)
nav.appendChild(button)

const applyToggle = (list, button, breakpoint) => {
  const navList = document.querySelector(list)
  if (document.body.clientWidth < breakpoint) {
    const navHeight = navList.clientHeight
    const navButton = document.querySelector(button)
    navList.style.maxHeight = '0px'

    navButton.onclick = () => {
      if (navList.style.maxHeight == '0px') {
        navList.style.maxHeight = `${navHeight}px`
      } else {
        navList.style.maxHeight = '0px'
      }
    }
  }
}

applyToggle('.list--nav', '.button', 640)

</script>

  </div>

  






</header>


<main class="main  container">

	<article class="article  article--page  content  typeset">

    <h1>Projects</h1>

    <h4 id="short-read-sequencing"><strong>Short read sequencing</strong></h4>

<p>Over the last decade, technology advances have made DNA sequencing a routine and cost-effective method in many fields of life sciences research. The dominant technology today generates billions of short sequences, called “reads”, consisting of 75-250 bases (the letters that make up the DNA sequence). We build high throughput analysis methods to process large volumes of reads in diverse DNA sequencing projects, from high profile international cancer genome mapping initiatives to the generation of reference genomes of non-model species.</p>

<h4 id="long-read-sequencing"><strong>Long read sequencing</strong></h4>

<p>New technologies are now becoming available that generate information on long stretches of the input DNA as long or linked reads. Long read platforms can sequence over 100,000 base pairs per read, though with a very high error rate and low throughput. Linked read platforms can associate multiple reads over similar lengths, although the data contains many gaps. Still, if coupled with bioinformatics tools that can leverage the rich information they provide, these new sequencing platforms will open new frontiers in health research. We develop specialized tools that quickly, accurately, and efficiently map, assemble and analyse long and linked sequence reads.</p>

<h4 id="spruce-genomics"><strong>Spruce genomics</strong></h4>

<p>Spruce trees are Canada’s most significant forest resource. Spruces produce high quality wood and fibre that is widely used in the industry, and as dominant species of Canada’s forests, they provide essential local and global ecosystem services. We are collaborating with multiple organizations across Canada to build genomic resources for the species.</p>

<h4 id="antimicrobial-peptide-discovery"><strong>Antimicrobial peptide discovery</strong></h4>

<p>Bacteria can rapidly evolve to develop resistance to antibiotics, presenting a growing and very dangerous problem. In a “post-antibiotic era”, bacterial diseases would once again be untreatable, and many standard treatments, including surgical operations, could become unusable. To boost the search for new treatment options, in a collaborative project, we are focusing on short proteins called antimicrobial peptides (AMPs), which are produced naturally by various animal and plant species. These host defence proteins can protect against infection, or reduce the harm caused by an existing infection.</p>

<h4 id="alternative-polyadenylation"><strong>Alternative polyadenylation</strong></h4>

<p>The length of a coding transcript’s tail end (3’ UTR) can affect its stability, transport and translation, with important regulatory consequences. Alternative cleavage of transcripts at the 3’ UTR end (alaternative polyadenylation) is a difficult to study phenomenon, often requiring specialized and relatively expensive experimental methods. As a solution to this problem, we are building bioinformatics tools using standard transcriptome sequencing (RNA-seq) datasets. We are using our tools to catalogue alternative polyadenylation in large cancer cohorts.</p>

<h4 id="clinical-bioinformatics"><strong>Clinical bioinformatics</strong></h4>

<p>Substantial advancements in healthcare economics can be realized through the development of genomics technologies to detect variations and mutations in DNA and RNA in a manner that allows: i) effective preventative care, and/or ii) efficient diagnosis and treatment. One technology that will enable this vision is high throughput DNA and RNA sequencing. This requires proper downstream data analysis and interpretation. To address this challenge, we are building and validating bioinformatics pipelines for clinical use. In collaborative projects, we are deploying these pipelines for cancer care and for diagnosing rare genetic diseases.</p>


  </article>

	

</main>

<footer class="footer">
  <div class="container">
    <div class="copyright  typeset">
      <small class="small">© Bioinformatics Technology Lab 2018</small>
    </div>

    
<nav class="nav  nav--footer">
  <ul class="list list--nav">
    

      

      <li class="item  item--nav">
        <a href="https://github.com/daviddarnes/alembic">Alembic by David Darnes</a>
      </li>
    
  </ul>
</nav>


  </div>
</footer>



	</body>
</html>
