<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Bioinformatics Technology Lab</title>
  <meta name="description" content="Bioinformatics Technology Lab at Canada&#39;s Michael Smith Genome Sciences Centre. Creates software for de novo genome assembly.">

  <link rel="stylesheet" href="/assets/main.css">
  <link rel="canonical" href="http://localhost:4000/publications.html">
  <link rel="alternate" type="application/rss+xml" title="Bioinformatics Technology Lab" href="/feed.xml">
  
  
</head>


  <body>
	<img style="display:block; margin-left: auto; margin-right:auto; width:100%" src="images/btlheader.png">

    <header class="site-header" role="banner">

  <div class="wrapper">
    
    
    <a class="site-title" href="/">Bioinformatics Technology Lab</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
            <a class="page-link" href="/members.html">Members</a>
            
          
            
            
            <a class="page-link" href="/projects.html">Active Projects</a>
            
          
            
            
            <a class="page-link" href="/software.html">Software</a>
            
          
            
            
            <a class="page-link" href="/news.html">News</a>
            
          
            
            
          
        </div>
      </nav>
    
  </div>
</header>




    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <header class="post-header">
    <h1 class="post-title"></h1>
  </header>

  <div class="post-content">
    <p>H Khan, H Mohamadi, BP Vandervalk, RL Warren, J Chu, <strong>I Birol</strong>. ChopStitch: exon annotation and splice graph construction using transcriptome assembly and whole genome sequencing data. 2017 Dec 29. <a href="https://academic.oup.com/bioinformatics/advance-article-pdf/doi/10.1093/bioinformatics/btx839/23166123/btx839.pdf"><em>Bioinformatics</em></a></p>

<p>SJ Jones, M Haulena, GA Taylor, S Chan, S Bilobram, RL Warren. The Genome of the Northern Sea Otter (Enhydra lutris kenyoni). 2017 Dec 11. <a href="http://www.mdpi.com/2073-4425/8/12/379/html"><em>Genes 8 (12), 379</em></a></p>

<p>SJM Jones, GA Taylor, S Chan, RL Warren, SA Hammond, S Bilobram. The Genome of the Beluga Whale (Delphinapterus leucas). 2017 Dec 11. <a href="http://www.mdpi.com/2073-4425/8/12/378/htm"><em>Genes 8 (12), 378</em></a></p>

<p>SA Hammond, RL Warren, BP Vandervalk, E Kucuk, H Khan, EA Gibb. The North American bullfrog draft genome provides insight into hormonal regulation of long noncoding RNA. 2017 Nov 10. <a href="https://www.nature.com/articles/s41467-017-01316-7"><em>Nature Communications 8 (1), 1433</em></a></p>

<p>S Yeo, L Coombe, RL Warren, J Chu, <strong>I Birol</strong>. ARCS: scaffolding genome drafts with linked reads. 2017 Oct 23. <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btx675/4562503"><em>Bioinformatics</em></a></p>

<p>AG Robertson, J Kim, H Al-Ahmadie, J Bellmunt, G Guo, AD Cherniack. Comprehensive molecular characterization of muscle-invasive bladder cancer. 2017 Oct 19. <a href="https://www.sciencedirect.com/science/article/pii/S0092867417310565"><em>Cell 171 (3), 540-556. e25</em></a></p>

<p>M England, H Errami, D Grigoriev, O Radulescu, T Sturm, A Weber. Symbolic Versus Numerical Computation and Visualization of Parameter Regions for Multistationarity of Biological Networks. 2017 Sep 18. <a href="https://link.springer.com/chapter/10.1007/978-3-319-66320-3_8"><em>International Workshop on Computer Algebra in Scientific Computing, 93-108</em></a></p>

<p>NA Hasan, RL Warren, LE Epperson, A Malecha, DC Alexander. Complete Genome Sequence of Mycobacterium chimaera SJ42, a Nonoutbreak Strain from an Immunocompromised Patient with Pulmonary Disease. 2017 Sep 14. <a href="http://genomea.asm.org/content/5/37/e00963-17.full"><em>Genome announcements 5 (37), e00963-17</em></a></p>

<p>AG Robertson, J Shih, C Yau, EA Gibb, J Oba, KL Mungall, JM Hess. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. 2017 Aug 14. <a href="https://www.sciencedirect.com/science/article/pii/S1535610817302957"><em>Cancer cell 32 (2), 204-220. e15</em></a></p>

<p>BJ Raphael, RH Hruban, AJ Aguirre, RA Moffitt, JJ Yeh, C Stewart. Integrated genomic characterization of pancreatic ductal adenocarcinoma. 2017 Aug 14. <a href="https://www.sciencedirect.com/science/article/pii/S1535610817302994"><em>Cancer cell 32 (2), 185-203. e13</em></a></p>

<p>SD Jackman, BP Vandervalk, H Mohamadi, J Chu, S Yeo, SA Hammond. ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter. 2017 May 01. <a href="http://genome.cshlp.org/content/27/5/768.full"><em>Genome research 27 (5), 768-777</em></a></p>

<p>TM Brown, SA Hammond, B Behsaz, N Veldhoen, <strong>I Birol</strong>, CC Helbing. De novo assembly of the ringed seal (Pusa hispida) blubber transcriptome: A tool that enables identification of molecular health indicators associated with PCB expo…. 2017 Apr 30. <a href="http://www.sciencedirect.com/science/article/pii/S0166445X17300358"><em>Aquatic Toxicology 185, 48-57</em></a></p>

<p>N Pavy, M Lamothe, B Pelgas, F Gagnon, <strong>I Birol</strong>, J Bohlmann, J Mackay. A high‐resolution reference genetic map positioning 8.8 K genes for the conifer white spruce: structural genomics implications and correspondence with physical dis…. 2017 Apr 01. <a href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.13478/full"><em>The Plant Journal 90 (1), 189-203</em></a></p>

<p>C Yang, J Chu, RL Warren, <strong>I Birol</strong>. NanoSim: nanopore sequence read simulator based on statistical characterization. 2017 Feb 24. <a href="https://academic.oup.com/gigascience/article/3051934"><em>GigaScience 6 (4), 1-6</em></a></p>

<p>E Kucuk, J Chu, BP Vandervalk, SA Hammond, RL Warren, <strong>I Birol</strong>. Kollector: transcript-informed, targeted de novo assembly of gene loci. 2017 Feb 10. <a href="https://academic.oup.com/bioinformatics/article/2982053"><em>Bioinformatics, btx078</em></a></p>

<p>H Mohamadi, H Khan, <strong>I Birol</strong>. ntCard: a streaming algorithm for cardinality estimation in genomics data. 2017 Jan 05. <a href="https://academic.oup.com/bioinformatics/article/2832780"><em>Bioinformatics 33 (9), 1324-1330</em></a></p>

<p>ML Thibodeau, M Steinraths, L Brown, Z Zong, N Shomer, S Taubert. Genomic and Cytogenetic Characterization of a Balanced Translocation Disrupting NUP98. 2017. <a href="http://gw2jh3xr2c.scholar.serialssolutions.com/?sid=google&amp;auinit=ML&amp;aulast=Thibodeau&amp;atitle=Genomic+and+Cytogenetic+Characterization+of+a+Balanced+Translocation+Disrupting+NUP98&amp;id=pmid:28854430"><em>Cytogenetic and genome research 152 (3), 117-121</em></a></p>

<p>Z Xue, RL Warren, EA Gibb, D MacMillan, J Wong, R Chiu, SA Hammond. Pan-cancer analysis reveals complex tumor-specific alternative polyadenylation. 2017 Jan 01. <a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/08/160960.full.pdf"><em>bioRxiv, 160960</em></a></p>

<p>S Yeo, L Coombe, J Chu, RL Warren, <strong>I Birol</strong>. ARCS: assembly roundup by chromium scaffolding. 2017 Jan 01. <a href="http://biorxiv.org/content/biorxiv/early/2017/01/17/100750.full.pdf"><em>bioRxiv, 100750</em></a></p>

<p>SA Hammond, RL Warren, BP Vandervalk, E Kucuk, H Khan, EA Gibb. It’s okay to be green: Draft genome of the North American bullfrog (Rana [Lithobates] catesbeiana). 2017 Jan 01. <a href="https://www.biorxiv.org/content/biorxiv/early/2017/01/13/100149.full.pdf"><em>bioRxiv, 100149</em></a></p>

<p>M Krzywinski, KM Nip, <strong>I Birol</strong>, M Marra. Differential Hive Plots: Seeing Networks Change. 2017. <a href="https://muse.jhu.edu/article/670962"><em>Leonardo 50 (5), 504-504</em></a></p>

<p>N Feau, G Taylor, AL Dale, B Dhillon, GJ Bilodeau, <strong>I Birol</strong>, SJM Jones. Genome sequences of six Phytophthora species threatening forest ecosystems. 2016 Dec 31. <a href="http://www.sciencedirect.com/science/article/pii/S2213596016301283"><em>Genomics data 10, 85-88</em></a></p>

<p>J Chu, H Mohamadi, RL Warren, C Yang, <strong>I Birol</strong>. Innovations and challenges in detecting long read overlaps: an evaluation of the state-of-the-art. 2016 Dec 21. <a href="https://academic.oup.com/bioinformatics/article/33/8/1261/2730233"><em>Bioinformatics 33 (8), 1261-1270</em></a></p>

<p>L Coombe, RL Warren, SD Jackman, C Yang, BP Vandervalk, RA Moore. Assembly of the complete Sitka spruce chloroplast genome using 10X Genomics’ GemCode sequencing data. 2016 Sep 15. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0163059"><em>PloS one 11 (9), e0163059</em></a></p>

<p>G Sheikhshab, E Starks, A Karsan, A Sarkar, <strong>I Birol</strong>. Graph-based Semi-supervised Gene Mention Tagging. 2016 Aug 12. <a href="https://www.researchgate.net/profile/Chaitanya_Shivade/publication/306099600_Identification_characterization_and_grounding_of_gradable_terms_in_clinical_text/links/57c84da108aec24de044f027.pdf#page=39"><em>ACL 2016, 27</em></a></p>


  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Bioinformatics Technology Lab</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            
              Bioinformatics Technology Lab
            
            </li>
            
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/bcgsc"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">bcgsc</span></a>

          </li>
          

          
        </ul>
      </div>

      <div class="footer-col footer-col-3">
        <p>Bioinformatics Technology Lab at Canada&#39;s Michael Smith Genome Sciences Centre. Creates software for de novo genome assembly.</p>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
