<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Bioinformatics Technology Lab</title>
  <meta name="description" content="Bioinformatics Technology Lab at Canada&#39;s Michael Smith Genome Sciences Centre. Creates software for de novo genome assembly.">

  <link rel="stylesheet" href="/assets/main.css">
  <link rel="canonical" href="http://localhost:4000/publications.html">
  <link rel="alternate" type="application/rss+xml" title="Bioinformatics Technology Lab" href="/feed.xml">
  
  
</head>


  <body>
	<img style="display:block; margin-left: auto; margin-right:auto; width:100%" src="../images/btlheader.png">

    <header class="site-header" role="banner">

  <div class="wrapper">
    
    
    <a class="site-title" href="/">Bioinformatics Technology Lab</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
            <a class="page-link" href="/members.html">Members</a>
            
          
            
            
            <a class="page-link" href="/projects.html">Active Projects</a>
            
          
            
            
            <a class="page-link" href="/software.html">Software</a>
            
          
            
            
            <a class="page-link" href="/news.html">News</a>
            
          
        </div>
      </nav>
    
  </div>
</header>




    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <header class="post-header">
    <h1 class="post-title"></h1>
  </header>

  <div class="post-content">
    <p>H Khan, H Mohamadi, BP Vandervalk, RL Warren, J Chu, <strong>I Birol</strong>. ChopStitch: exon annotation and splice graph construction using transcriptome assembly and whole genome sequencing data. . Bioinformatics<br />
SJ Jones, M Haulena, GA Taylor, S Chan, S Bilobram, RL Warren. The Genome of the Northern Sea Otter (Enhydra lutris kenyoni). . Genes 8 (12), 379<br />
SJM Jones, GA Taylor, S Chan, RL Warren, SA Hammond, S Bilobram. The Genome of the Beluga Whale (Delphinapterus leucas). . Genes 8 (12), 378<br />
SA Hammond, RL Warren, BP Vandervalk, E Kucuk, H Khan, EA Gibb. The North American bullfrog draft genome provides insight into hormonal regulation of long noncoding RNA. . Nature Communications 8 (1), 1433<br />
S Yeo, L Coombe, RL Warren, J Chu, <strong>I Birol</strong>. ARCS: scaffolding genome drafts with linked reads. . Bioinformatics<br />
AG Robertson, J Kim, H Al-Ahmadie, J Bellmunt, G Guo, AD Cherniack. Comprehensive molecular characterization of muscle-invasive bladder cancer. . Cell 171 (3), 540-556. e25<br />
M England, H Errami, D Grigoriev, O Radulescu, T Sturm, A Weber. Symbolic Versus Numerical Computation and Visualization of Parameter Regions for Multistationarity of Biological Networks. . International Workshop on Computer Algebra in Scientific Computing, 93-108<br />
NA Hasan, RL Warren, LE Epperson, A Malecha, DC Alexander. Complete Genome Sequence of Mycobacterium chimaera SJ42, a Nonoutbreak Strain from an Immunocompromised Patient with Pulmonary Disease. . Genome announcements 5 (37), e00963-17<br />
AG Robertson, J Shih, C Yau, EA Gibb, J Oba, KL Mungall, JM Hess. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. . Cancer cell 32 (2), 204-220. e15<br />
BJ Raphael, RH Hruban, AJ Aguirre, RA Moffitt, JJ Yeh, C Stewart. Integrated genomic characterization of pancreatic ductal adenocarcinoma. . Cancer cell 32 (2), 185-203. e13<br />
SD Jackman, BP Vandervalk, H Mohamadi, J Chu, S Yeo, SA Hammond. ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter. . Genome research 27 (5), 768-777<br />
TM Brown, SA Hammond, B Behsaz, N Veldhoen, <strong>I Birol</strong>, CC Helbing. De novo assembly of the ringed seal (Pusa hispida) blubber transcriptome: A tool that enables identification of molecular health indicators associated with PCB expo…. . Aquatic Toxicology 185, 48-57<br />
N Pavy, M Lamothe, B Pelgas, F Gagnon, <strong>I Birol</strong>, J Bohlmann, J Mackay. A high‐resolution reference genetic map positioning 8.8 K genes for the conifer white spruce: structural genomics implications and correspondence with physical dis…. . The Plant Journal 90 (1), 189-203<br />
C Yang, J Chu, RL Warren, <strong>I Birol</strong>. NanoSim: nanopore sequence read simulator based on statistical characterization. . GigaScience 6 (4), 1-6<br />
E Kucuk, J Chu, BP Vandervalk, SA Hammond, RL Warren, <strong>I Birol</strong>. Kollector: transcript-informed, targeted de novo assembly of gene loci. . Bioinformatics, btx078<br />
H Mohamadi, H Khan, <strong>I Birol</strong>. ntCard: a streaming algorithm for cardinality estimation in genomics data. . Bioinformatics 33 (9), 1324-1330<br />
ML Thibodeau, M Steinraths, L Brown, Z Zong, N Shomer, S Taubert. Genomic and Cytogenetic Characterization of a Balanced Translocation Disrupting NUP98. . Cytogenetic and genome research 152 (3), 117-121<br />
Z Xue, RL Warren, EA Gibb, D MacMillan, J Wong, R Chiu, SA Hammond. Pan-cancer analysis reveals complex tumor-specific alternative polyadenylation. . bioRxiv, 160960<br />
S Yeo, L Coombe, J Chu, RL Warren, <strong>I Birol</strong>. ARCS: assembly roundup by chromium scaffolding. . bioRxiv, 100750<br />
SA Hammond, RL Warren, BP Vandervalk, E Kucuk, H Khan, EA Gibb. It’s okay to be green: Draft genome of the North American bullfrog (Rana [Lithobates] catesbeiana). . bioRxiv, 100149<br />
M Krzywinski, KM Nip, <strong>I Birol</strong>, M Marra. Differential Hive Plots: Seeing Networks Change. . Leonardo 50 (5), 504-504<br />
N Feau, G Taylor, AL Dale, B Dhillon, GJ Bilodeau, <strong>I Birol</strong>, SJM Jones. Genome sequences of six Phytophthora species threatening forest ecosystems. . Genomics data 10, 85-88<br />
J Chu, H Mohamadi, RL Warren, C Yang, <strong>I Birol</strong>. Innovations and challenges in detecting long read overlaps: an evaluation of the state-of-the-art. . Bioinformatics 33 (8), 1261-1270<br />
L Coombe, RL Warren, SD Jackman, C Yang, BP Vandervalk, RA Moore. Assembly of the complete Sitka spruce chloroplast genome using 10X Genomics’ GemCode sequencing data. . PloS one 11 (9), e0163059<br />
G Sheikhshab, E Starks, A Karsan, A Sarkar, <strong>I Birol</strong>. Graph-based Semi-supervised Gene Mention Tagging. . ACL 2016, 27<br />
D Bouneffouf, <strong>I Birol</strong>. Theoretical analysis of the Minimum Sum of Squared Similarities sampling for Nyström-based spectral clustering. . Neural Networks (IJCNN), 2016 International Joint Conference on, 3856-3862<br />
D Bouneffouf, <strong>I Birol</strong>. Ensemble Minimum Sum of Squared Similarities sampling for Nyström-based spectral clustering. . Neural Networks (IJCNN), 2016 International Joint Conference on, 3851-3855<br />
H Mohamadi, J Chu, BP Vandervalk, <strong>I Birol</strong>. ntHash: recursive nucleotide hashing. . Bioinformatics 32 (22), 3492-3494<br />
AS Morrissy, L Garzia, DJH Shih, S Zuyderduyn, X Huang, P Skowron. Divergent clonal selection dominates medulloblastoma at recurrence. . Nature 529 (7586), 351-357<br />
A Chu, G Robertson, D Brooks, AJ Mungall, <strong>I Birol</strong>, R Coope, Y Ma. Large-scale profiling of microRNAs for the cancer genome atlas. . Nucleic acids research 44 (1), e3-e3<br />
S Jackman, <strong>I Birol</strong>, S Jackman, <strong>I Birol</strong>. Linuxbrew and homebrew for cross-platform package management. . F1000Research 5, 1795<br />
J Chu, H Mohamadi, R Warren, C Yang, <strong>I Birol</strong>. Overlapping long sequence reads: Current innovations and challenges in developing sensitive, specific and scalable algorithms. . bioRxiv, 081596<br />
SD Jackman, RL Warren, EA Gibb, BP Vandervalk, H Mohamadi, J Chu. Organellar genomes of white spruce (Picea glauca): assembly and annotation. . Genome biology and evolution 8 (1), 29-41<br />
H Lin, J Zeng, K Rothe, J Ruschmann, O Petriv, K O’Neill, T Maetzig. Identification and Characterization of New Microrna Biomarkers and Target Genes in Drug-Insensitive CD34+ CML Stem/Progenitor Cells Using Global Comparative…. . Blood 126 (23), 1567-1567<br />
<strong>I Birol</strong>, J Chu, H Mohamadi, SD Jackman, K Raghavan, BP Vandervalk. Spaced seed data structures for de novo assembly. . International journal of genomics 2015<br />
BP Vandervalk, C Yang, Z Xue, K Raghavan, J Chu, H Mohamadi. Konnector v2. 0: pseudo-long reads from paired-end sequencing data. . BMC medical genomics 8 (3), S1<br />
H Lin, K Rothe, J Ruschmann, O Petriv, K O’Neill, D Knapp, R Brinkman. Comparative RNA-seq analyses identify new microRNA biomarkers and candidate target genes in patients’ drug-resistant CD34+ CML stem/progenitor cells. . Experimental Hematology 43 (9), S63<br />
RL Warren, C Yang, BP Vandervalk, B Behsaz, A Lagman, SJM Jones. LINKS: Scalable, alignment-free scaffolding of draft genomes with long reads. . GigaScience 4 (1), 35<br />
D Paulino, RL Warren, BP Vandervalk, A Raymond, SD Jackman, <strong>I Birol</strong>. Sealer: a scalable gap-closing application for finishing draft genomes. . BMC bioinformatics 16 (1), 230<br />
D Bouneffouf, <strong>I Birol</strong>. Sampling with Minimum Sum of Squared Similarities for Nystrom-Based Large Scale Spectral Clustering.. . IJCAI, 2313-2319<br />
RL Warren, CI Keeling, MMS Yuen, A Raymond, GA Taylor. Improved white spruce (Picea glauca) genome assemblies and annotation of large gene families of conifer terpenoid and phenolic defense metabolism. . The Plant Journal 83 (2), 189-212<br />
<strong>I Birol</strong>, B Behsaz, SA Hammond, E Kucuk, N Veldhoen, CC Helbing. De novo transcriptome assemblies of Rana (Lithobates) catesbeiana and Xenopus laevis tadpole livers for comparative genomics without reference genomes. . PloS one 10 (6), e0130720<br />
SD Jackman, J Bohlmann, <strong>I Birol</strong>. UniqTag: Content-derived unique and stable identifiers for gene annotation. . PloS one 10 (5), e0128026<br />
H Mohamadi, BP Vandervalk, A Raymond, SD Jackman, J Chu. DIDA: Distributed Indexing Dispatched Alignment. . PloS one 10 (4), e0126409<br />
E Shoshan, AK Mobley, RR Braeuer, T Kamiya, L Huang, ME Vasquez. Reduced adenosine-to-inosine miR-455-5p editing promotes melanoma growth and metastasis. . Nature cell biology 17 (3), 311-321<br />
ARMA Basher, AS Purdy, <strong>I Birol</strong>. Event Extraction from Biomedical Literature. . bioRxiv, 034397<br />
RL Warren, BP Vandervalk, SJM Jones, <strong>I Birol</strong>. LINKS: Scaffolding genome assemblies with kilobase-long nanopore reads. . bioRxiv, 016519<br />
H Lin, K Rothe, J Ruschmann, O Petriv, K O’Neill, DJHF Knapp. Identification of new microRNA biomarkers and candidate target genes in primitive CML cells using global comparative RNA analyses. . Blood 124 (21), 3133-3133<br />
R Amanda, <strong>I Birol</strong>, J Bousquet, PK Ingvarsson, S Jansson, SJM Jones. Insights into conifer giga-genomes. . Plant physiology 166 (4), 1724-1732<br />
<strong>I Birol</strong>, H Mohamadi, A Raymond, K Raghavan, J Chu, BP Vandervalk. Spaced seed data structures. . Bioinformatics and Biomedicine (BIBM), 2014 IEEE International Conference on …<br />
BP Vandervalk, SD Jackman, A Raymond, H Mohamadi, C Yang. Konnector: Connecting paired-end reads using a bloom filter de Bruijn graph. . Bioinformatics and Biomedicine (BIBM), 2014 IEEE International Conference on …<br />
M Parfenov, CS Pedamallu, N Gehlenborg, SS Freeman, L Danilova. Characterization of HPV and host genome interactions in primary head and neck cancers. . Proceedings of the National Academy of Sciences 111 (43), 15544-15549<br />
Cancer Genome Atlas Research Network. Comprehensive molecular characterization of gastric adenocarcinoma. . Nature 513 (7517), 202-209<br />
J Chu, S Sadeghi, A Raymond, SD Jackman, KM Nip, R Mar. BioBloom tools: fast, accurate and memory-efficient host species sequence screening using bloom filters. . Bioinformatics 30 (23), 3402-3404<br />
YS Butterfield, M Kreitzman, N Thiessen, RD Corbett, Y Li, J Pang, YP Ma. JAGuaR: junction alignments to genome for RNA-seq reads. . PloS one 9 (7), e102398<br />
JS Sena, I Giguère, B Boyle, P Rigault, <strong>I Birol</strong>, A Zuccolo, K Ritland. Evolution of gene structure in the conifer Picea glauca: a comparative analysis of the impact of intron size. . BMC plant biology 14 (1), 95<br />
<strong>I Birol</strong>, A Raymond, R Chiu, KM Nip, SD Jackman, M Kreitzman. Kleat: cleavage site analysis of transcriptomes. . Pacific Symposium on Biocomputing Co-Chairs, 347-358<br />
IE Bosdet, TR Docking, YS Butterfield, AJ Mungall, T Zeng, RJ Coope. A clinically validated diagnostic second-generation sequencing assay for detection of hereditary BRCA1 and BRCA2 mutations. . The Journal of Molecular Diagnostics 15 (6), 796-809<br />
JN Weinstein, EA Collisson, GB Mills, KRM Shaw, BA Ozenberger. The cancer genome atlas pan-cancer analysis project. . Nature genetics 45 (10), 1113-1120<br />
M Lenassi, C Gostincar, S Jackman, M Turk, I Sadowski, C Nislow. Whole genome duplication and enrichment of cation transporters are associated with the halotolerant lifestyle of the black yeast Hortaea werneckii. . YEAST 30, 190-190<br />
M Lenassi, C Gostinčar, S Jackman, M Turk, I Sadowski, C Nislow. Whole genome duplication and enrichment of metal cation transporters revealed by de novo genome sequencing of extremely halotolerant black yeast Hortaea wer…. . PLoS One 8 (8), e71328<br />
RD Morin, K Mungall, E Pleasance, AJ Mungall, R Goya, RD Huff. Mutational and structural analysis of diffuse large B-cell lymphoma using whole-genome sequencing. . Blood 122 (7), 1256-1265<br />
L Swanson, G Robertson, KL Mungall, YS Butterfield, R Chiu, RD Corbett. Barnacle: detecting and characterizing tandem duplications and fusions in transcriptome assemblies. . BMC genomics 14 (1), 550<br />
A Fentiman, J Parker, L Swanson, K Mungall, <strong>I Birol</strong>, A Karsan. Endothelial-hematopoietic transition: from the dorsal aorta to the dish. . Experimental Hematology 41 (8), S26<br />
KR Bradnam, JN Fass, A Alexandrov, P Baranay, M Bechner, <strong>I Birol</strong>. Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species. . GigaScience 2 (1), 10<br />
K Kasaian, SM Wiseman, N Thiessen, KL Mungall, RD Corbett, JQ Qian. Complete genomic landscape of a recurring sporadic parathyroid carcinoma. . The Journal of pathology 230 (3), 249-260<br />
S Haridas, Y Wang, L Lim, SM Alamouti, S Jackman, R Docking. The genome and transcriptome of the pine saprophyte Ophiostoma piceae, and a comparison with the bark beetle-associated pine pathogen Grosmannia clavigera. . BMC genomics 14 (1), 373<br />
Cancer Genome Atlas Research Network. Genomic and epigenomic landscapes of adult de novo acute myeloid leukemia. . N Engl j Med 2013 (368), 2059-2074<br />
AH Khodabakhshi, AP Fejes, <strong>I Birol</strong>, SJM Jones. Identifying cancer mutation targets across thousands of samples: MuteProc, a high throughput mutation analysis pipeline. . BMC bioinformatics 14 (1), 167<br />
<strong>I Birol</strong>, A Raymond, SD Jackman, S Pleasance, R Coope, GA Taylor. Assembling the 20 Gb white spruce (Picea glauca) genome from whole-genome shotgun sequencing data. . Bioinformatics 29 (12), 1492-1497<br />
Cancer Genome Atlas Research Network. Integrated genomic characterization of endometrial carcinoma. . Nature 497 (7447), 67<br />
CI Keeling, MMS Yuen, NY Liao, TR Docking, SK Chan, GA Taylor. Draft genome of the mountain pine beetle, Dendroctonus ponderosae Hopkins, a major forest pest. . Genome biology 14 (3), R27<br />
TJ Pugh, O Morozova, EF Attiyeh, S Asgharzadeh, JS Wei, D Auclair. The genetic landscape of high-risk neuroblastoma. . Nature genetics 45 (3), 279-284<br />
AH Khodabakhshi, RD Morin, AP Fejes, AJ Mungall, KL Mungall. Recurrent targets of aberrant somatic hypermutation in lymphoma. . Oncotarget 3 (11), 1308<br />
Cancer Genome Atlas Network. Comprehensive molecular portraits of human breast tumours. . Nature 490 (7418), 61-70<br />
K Kasaian, A He, N Thiessen, KL Mungall, J Qian, R Varhol, Y Zhao. Profiling thyroid cancers on the molecular level. . Genome 55 (10), 705-705<br />
Cancer Genome Atlas Research Network. Comprehensive genomic characterization of squamous cell lung cancers. . Nature 489 (7417), 519-525<br />
CI Keeling, H Henderson, M Li, M Yuen, EL Clark, JD Fraser, DPW Huber. Transcriptome and full-length cDNA resources for the mountain pine beetle, Dendroctonus ponderosae Hopkins, a major insect pest of pine forests. . Insect biochemistry and molecular biology 42 (8), 525-536<br />
KG Roberts, RD Morin, J Zhang, M Hirst, Y Zhao, X Su, SC Chen. Genetic alterations activating kinase and cytokine receptor signaling in high-risk acute lymphoblastic leukemia. . Cancer cell 22 (2), 153-166<br />
PA Northcott, DJH Shih, J Peacock, L Garzia, AS Morrissy, T Zichner. Subgroup-specific structural variation across 1,000 medulloblastoma genomes. . Nature 488 (7409), 49-56<br />
Cancer Genome Atlas Network. Comprehensive molecular characterization of human colon and rectal cancer. . Nature 487 (7407), 330-337<br />
SP Shah, A Roth, R Goya, A Oloumi, G Ha, Y Zhao, G Turashvili, J Ding. The clonal and mutational evolution spectrum of primary triple-negative breast cancers. . Nature 486 (7403), 395-399<br />
A Hadj Khodabakhshi, RD Morin, AP Fejes, AJ Mungall, KL Mungall. Recurrent targets of aberrant somatic hypermutation in lymphoma. . Oncotarget 3 (11)<br />
D Yorukoglu, F Hach, L Swanson, CC Collins, <strong>I Birol</strong>, SC Sahinalp. Dissect: detection and characterization of novel structural alterations in transcribed sequences. . Bioinformatics 28 (12), i179-i187<br />
M Sze, PA Dimitriu, JV Gosselink, WM Elliott, MM Moore, S Adams. A Preliminary Assessment Of Fungi In The Lung Microbiome. . C52. COPD PATHOGENESIS: HUMAN STUDIES, A4537-A4537</p>

  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Bioinformatics Technology Lab</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            
              Bioinformatics Technology Lab
            
            </li>
            
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/bcgsc"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">bcgsc</span></a>

          </li>
          

          
        </ul>
      </div>

      <div class="footer-col footer-col-3">
        <p>Bioinformatics Technology Lab at Canada&#39;s Michael Smith Genome Sciences Centre. Creates software for de novo genome assembly.</p>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
