H Khan, H Mohamadi, BP Vandervalk, RL Warren, J Chu, **I Birol**. ChopStitch: exon annotation and splice graph construction using transcriptome assembly and whole genome sequencing data. . Bioinformatics  
SJ Jones, M Haulena, GA Taylor, S Chan, S Bilobram, RL Warren. The Genome of the Northern Sea Otter (Enhydra lutris kenyoni). . Genes 8 (12), 379  
SJM Jones, GA Taylor, S Chan, RL Warren, SA Hammond, S Bilobram. The Genome of the Beluga Whale (Delphinapterus leucas). . Genes 8 (12), 378  
SA Hammond, RL Warren, BP Vandervalk, E Kucuk, H Khan, EA Gibb. The North American bullfrog draft genome provides insight into hormonal regulation of long noncoding RNA. . Nature Communications 8 (1), 1433  
S Yeo, L Coombe, RL Warren, J Chu, **I Birol**. ARCS: scaffolding genome drafts with linked reads. . Bioinformatics  
AG Robertson, J Kim, H Al-Ahmadie, J Bellmunt, G Guo, AD Cherniack. Comprehensive molecular characterization of muscle-invasive bladder cancer. . Cell 171 (3), 540-556. e25  
M England, H Errami, D Grigoriev, O Radulescu, T Sturm, A Weber. Symbolic Versus Numerical Computation and Visualization of Parameter Regions for Multistationarity of Biological Networks. . International Workshop on Computer Algebra in Scientific Computing, 93-108  
NA Hasan, RL Warren, LE Epperson, A Malecha, DC Alexander. Complete Genome Sequence of Mycobacterium chimaera SJ42, a Nonoutbreak Strain from an Immunocompromised Patient with Pulmonary Disease. . Genome announcements 5 (37), e00963-17  
AG Robertson, J Shih, C Yau, EA Gibb, J Oba, KL Mungall, JM Hess. Integrative analysis identifies four molecular and clinical subsets in uveal melanoma. . Cancer cell 32 (2), 204-220. e15  
BJ Raphael, RH Hruban, AJ Aguirre, RA Moffitt, JJ Yeh, C Stewart. Integrated genomic characterization of pancreatic ductal adenocarcinoma. . Cancer cell 32 (2), 185-203. e13  
SD Jackman, BP Vandervalk, H Mohamadi, J Chu, S Yeo, SA Hammond. ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter. . Genome research 27 (5), 768-777  
TM Brown, SA Hammond, B Behsaz, N Veldhoen, **I Birol**, CC Helbing. De novo assembly of the ringed seal (Pusa hispida) blubber transcriptome: A tool that enables identification of molecular health indicators associated with PCB expo.... . Aquatic Toxicology 185, 48-57  
N Pavy, M Lamothe, B Pelgas, F Gagnon, **I Birol**, J Bohlmann, J Mackay. A high‐resolution reference genetic map positioning 8.8 K genes for the conifer white spruce: structural genomics implications and correspondence with physical dis.... . The Plant Journal 90 (1), 189-203  
C Yang, J Chu, RL Warren, **I Birol**. NanoSim: nanopore sequence read simulator based on statistical characterization. . GigaScience 6 (4), 1-6  
E Kucuk, J Chu, BP Vandervalk, SA Hammond, RL Warren, **I Birol**. Kollector: transcript-informed, targeted de novo assembly of gene loci. . Bioinformatics, btx078  
H Mohamadi, H Khan, **I Birol**. ntCard: a streaming algorithm for cardinality estimation in genomics data. . Bioinformatics 33 (9), 1324-1330  
ML Thibodeau, M Steinraths, L Brown, Z Zong, N Shomer, S Taubert. Genomic and Cytogenetic Characterization of a Balanced Translocation Disrupting NUP98. . Cytogenetic and genome research 152 (3), 117-121  
Z Xue, RL Warren, EA Gibb, D MacMillan, J Wong, R Chiu, SA Hammond. Pan-cancer analysis reveals complex tumor-specific alternative polyadenylation. . bioRxiv, 160960  
S Yeo, L Coombe, J Chu, RL Warren, **I Birol**. ARCS: assembly roundup by chromium scaffolding. . bioRxiv, 100750  
SA Hammond, RL Warren, BP Vandervalk, E Kucuk, H Khan, EA Gibb. It's okay to be green: Draft genome of the North American bullfrog (Rana [Lithobates] catesbeiana). . bioRxiv, 100149  
M Krzywinski, KM Nip, **I Birol**, M Marra. Differential Hive Plots: Seeing Networks Change. . Leonardo 50 (5), 504-504  
N Feau, G Taylor, AL Dale, B Dhillon, GJ Bilodeau, **I Birol**, SJM Jones. Genome sequences of six Phytophthora species threatening forest ecosystems. . Genomics data 10, 85-88  
J Chu, H Mohamadi, RL Warren, C Yang, **I Birol**. Innovations and challenges in detecting long read overlaps: an evaluation of the state-of-the-art. . Bioinformatics 33 (8), 1261-1270  
L Coombe, RL Warren, SD Jackman, C Yang, BP Vandervalk, RA Moore. Assembly of the complete Sitka spruce chloroplast genome using 10X Genomics’ GemCode sequencing data. . PloS one 11 (9), e0163059  
G Sheikhshab, E Starks, A Karsan, A Sarkar, **I Birol**. Graph-based Semi-supervised Gene Mention Tagging. . ACL 2016, 27  
D Bouneffouf, **I Birol**. Theoretical analysis of the Minimum Sum of Squared Similarities sampling for Nyström-based spectral clustering. . Neural Networks (IJCNN), 2016 International Joint Conference on, 3856-3862  
D Bouneffouf, **I Birol**. Ensemble Minimum Sum of Squared Similarities sampling for Nyström-based spectral clustering. . Neural Networks (IJCNN), 2016 International Joint Conference on, 3851-3855  
H Mohamadi, J Chu, BP Vandervalk, **I Birol**. ntHash: recursive nucleotide hashing. . Bioinformatics 32 (22), 3492-3494  
AS Morrissy, L Garzia, DJH Shih, S Zuyderduyn, X Huang, P Skowron. Divergent clonal selection dominates medulloblastoma at recurrence. . Nature 529 (7586), 351-357  
A Chu, G Robertson, D Brooks, AJ Mungall, **I Birol**, R Coope, Y Ma. Large-scale profiling of microRNAs for the cancer genome atlas. . Nucleic acids research 44 (1), e3-e3  
S Jackman, **I Birol**, S Jackman, **I Birol**. Linuxbrew and homebrew for cross-platform package management. . F1000Research 5, 1795  
J Chu, H Mohamadi, R Warren, C Yang, **I Birol**. Overlapping long sequence reads: Current innovations and challenges in developing sensitive, specific and scalable algorithms. . bioRxiv, 081596  
SD Jackman, RL Warren, EA Gibb, BP Vandervalk, H Mohamadi, J Chu. Organellar genomes of white spruce (Picea glauca): assembly and annotation. . Genome biology and evolution 8 (1), 29-41  
H Lin, J Zeng, K Rothe, J Ruschmann, O Petriv, K O'Neill, T Maetzig. Identification and Characterization of New Microrna Biomarkers and Target Genes in Drug-Insensitive CD34+ CML Stem/Progenitor Cells Using Global Comparative.... . Blood 126 (23), 1567-1567  
**I Birol**, J Chu, H Mohamadi, SD Jackman, K Raghavan, BP Vandervalk. Spaced seed data structures for de novo assembly. . International journal of genomics 2015  
BP Vandervalk, C Yang, Z Xue, K Raghavan, J Chu, H Mohamadi. Konnector v2. 0: pseudo-long reads from paired-end sequencing data. . BMC medical genomics 8 (3), S1  
H Lin, K Rothe, J Ruschmann, O Petriv, K O'Neill, D Knapp, R Brinkman. Comparative RNA-seq analyses identify new microRNA biomarkers and candidate target genes in patients’ drug-resistant CD34+ CML stem/progenitor cells. . Experimental Hematology 43 (9), S63  
RL Warren, C Yang, BP Vandervalk, B Behsaz, A Lagman, SJM Jones. LINKS: Scalable, alignment-free scaffolding of draft genomes with long reads. . GigaScience 4 (1), 35  
D Paulino, RL Warren, BP Vandervalk, A Raymond, SD Jackman, **I Birol**. Sealer: a scalable gap-closing application for finishing draft genomes. . BMC bioinformatics 16 (1), 230  
D Bouneffouf, **I Birol**. Sampling with Minimum Sum of Squared Similarities for Nystrom-Based Large Scale Spectral Clustering.. . IJCAI, 2313-2319  
RL Warren, CI Keeling, MMS Yuen, A Raymond, GA Taylor. Improved white spruce (Picea glauca) genome assemblies and annotation of large gene families of conifer terpenoid and phenolic defense metabolism. . The Plant Journal 83 (2), 189-212  
**I Birol**, B Behsaz, SA Hammond, E Kucuk, N Veldhoen, CC Helbing. De novo transcriptome assemblies of Rana (Lithobates) catesbeiana and Xenopus laevis tadpole livers for comparative genomics without reference genomes. . PloS one 10 (6), e0130720  
SD Jackman, J Bohlmann, **I Birol**. UniqTag: Content-derived unique and stable identifiers for gene annotation. . PloS one 10 (5), e0128026  
H Mohamadi, BP Vandervalk, A Raymond, SD Jackman, J Chu. DIDA: Distributed Indexing Dispatched Alignment. . PloS one 10 (4), e0126409  
E Shoshan, AK Mobley, RR Braeuer, T Kamiya, L Huang, ME Vasquez. Reduced adenosine-to-inosine miR-455-5p editing promotes melanoma growth and metastasis. . Nature cell biology 17 (3), 311-321  
ARMA Basher, AS Purdy, **I Birol**. Event Extraction from Biomedical Literature. . bioRxiv, 034397  
RL Warren, BP Vandervalk, SJM Jones, **I Birol**. LINKS: Scaffolding genome assemblies with kilobase-long nanopore reads. . bioRxiv, 016519  
H Lin, K Rothe, J Ruschmann, O Petriv, K O'Neill, DJHF Knapp. Identification of new microRNA biomarkers and candidate target genes in primitive CML cells using global comparative RNA analyses. . Blood 124 (21), 3133-3133  
R Amanda, **I Birol**, J Bousquet, PK Ingvarsson, S Jansson, SJM Jones. Insights into conifer giga-genomes. . Plant physiology 166 (4), 1724-1732  
**I Birol**, H Mohamadi, A Raymond, K Raghavan, J Chu, BP Vandervalk. Spaced seed data structures. . Bioinformatics and Biomedicine (BIBM), 2014 IEEE International Conference on …  
BP Vandervalk, SD Jackman, A Raymond, H Mohamadi, C Yang. Konnector: Connecting paired-end reads using a bloom filter de Bruijn graph. . Bioinformatics and Biomedicine (BIBM), 2014 IEEE International Conference on …  
M Parfenov, CS Pedamallu, N Gehlenborg, SS Freeman, L Danilova. Characterization of HPV and host genome interactions in primary head and neck cancers. . Proceedings of the National Academy of Sciences 111 (43), 15544-15549  
Cancer Genome Atlas Research Network. Comprehensive molecular characterization of gastric adenocarcinoma. . Nature 513 (7517), 202-209  
J Chu, S Sadeghi, A Raymond, SD Jackman, KM Nip, R Mar. BioBloom tools: fast, accurate and memory-efficient host species sequence screening using bloom filters. . Bioinformatics 30 (23), 3402-3404  
YS Butterfield, M Kreitzman, N Thiessen, RD Corbett, Y Li, J Pang, YP Ma. JAGuaR: junction alignments to genome for RNA-seq reads. . PloS one 9 (7), e102398  
JS Sena, I Giguère, B Boyle, P Rigault, **I Birol**, A Zuccolo, K Ritland. Evolution of gene structure in the conifer Picea glauca: a comparative analysis of the impact of intron size. . BMC plant biology 14 (1), 95  
**I Birol**, A Raymond, R Chiu, KM Nip, SD Jackman, M Kreitzman. Kleat: cleavage site analysis of transcriptomes. . Pacific Symposium on Biocomputing Co-Chairs, 347-358  
IE Bosdet, TR Docking, YS Butterfield, AJ Mungall, T Zeng, RJ Coope. A clinically validated diagnostic second-generation sequencing assay for detection of hereditary BRCA1 and BRCA2 mutations. . The Journal of Molecular Diagnostics 15 (6), 796-809  
JN Weinstein, EA Collisson, GB Mills, KRM Shaw, BA Ozenberger. The cancer genome atlas pan-cancer analysis project. . Nature genetics 45 (10), 1113-1120  
M Lenassi, C Gostincar, S Jackman, M Turk, I Sadowski, C Nislow. Whole genome duplication and enrichment of cation transporters are associated with the halotolerant lifestyle of the black yeast Hortaea werneckii. . YEAST 30, 190-190  
M Lenassi, C Gostinčar, S Jackman, M Turk, I Sadowski, C Nislow. Whole genome duplication and enrichment of metal cation transporters revealed by de novo genome sequencing of extremely halotolerant black yeast Hortaea wer.... . PLoS One 8 (8), e71328  
RD Morin, K Mungall, E Pleasance, AJ Mungall, R Goya, RD Huff. Mutational and structural analysis of diffuse large B-cell lymphoma using whole-genome sequencing. . Blood 122 (7), 1256-1265  
L Swanson, G Robertson, KL Mungall, YS Butterfield, R Chiu, RD Corbett. Barnacle: detecting and characterizing tandem duplications and fusions in transcriptome assemblies. . BMC genomics 14 (1), 550  
A Fentiman, J Parker, L Swanson, K Mungall, **I Birol**, A Karsan. Endothelial-hematopoietic transition: from the dorsal aorta to the dish. . Experimental Hematology 41 (8), S26  
KR Bradnam, JN Fass, A Alexandrov, P Baranay, M Bechner, **I Birol**. Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species. . GigaScience 2 (1), 10  
K Kasaian, SM Wiseman, N Thiessen, KL Mungall, RD Corbett, JQ Qian. Complete genomic landscape of a recurring sporadic parathyroid carcinoma. . The Journal of pathology 230 (3), 249-260  
S Haridas, Y Wang, L Lim, SM Alamouti, S Jackman, R Docking. The genome and transcriptome of the pine saprophyte Ophiostoma piceae, and a comparison with the bark beetle-associated pine pathogen Grosmannia clavigera. . BMC genomics 14 (1), 373  
Cancer Genome Atlas Research Network. Genomic and epigenomic landscapes of adult de novo acute myeloid leukemia. . N Engl j Med 2013 (368), 2059-2074  
AH Khodabakhshi, AP Fejes, **I Birol**, SJM Jones. Identifying cancer mutation targets across thousands of samples: MuteProc, a high throughput mutation analysis pipeline. . BMC bioinformatics 14 (1), 167  
**I Birol**, A Raymond, SD Jackman, S Pleasance, R Coope, GA Taylor. Assembling the 20 Gb white spruce (Picea glauca) genome from whole-genome shotgun sequencing data. . Bioinformatics 29 (12), 1492-1497  
Cancer Genome Atlas Research Network. Integrated genomic characterization of endometrial carcinoma. . Nature 497 (7447), 67  
CI Keeling, MMS Yuen, NY Liao, TR Docking, SK Chan, GA Taylor. Draft genome of the mountain pine beetle, Dendroctonus ponderosae Hopkins, a major forest pest. . Genome biology 14 (3), R27  
TJ Pugh, O Morozova, EF Attiyeh, S Asgharzadeh, JS Wei, D Auclair. The genetic landscape of high-risk neuroblastoma. . Nature genetics 45 (3), 279-284  
AH Khodabakhshi, RD Morin, AP Fejes, AJ Mungall, KL Mungall. Recurrent targets of aberrant somatic hypermutation in lymphoma. . Oncotarget 3 (11), 1308  
Cancer Genome Atlas Network. Comprehensive molecular portraits of human breast tumours. . Nature 490 (7418), 61-70  
K Kasaian, A He, N Thiessen, KL Mungall, J Qian, R Varhol, Y Zhao. Profiling thyroid cancers on the molecular level. . Genome 55 (10), 705-705  
Cancer Genome Atlas Research Network. Comprehensive genomic characterization of squamous cell lung cancers. . Nature 489 (7417), 519-525  
CI Keeling, H Henderson, M Li, M Yuen, EL Clark, JD Fraser, DPW Huber. Transcriptome and full-length cDNA resources for the mountain pine beetle, Dendroctonus ponderosae Hopkins, a major insect pest of pine forests. . Insect biochemistry and molecular biology 42 (8), 525-536  
KG Roberts, RD Morin, J Zhang, M Hirst, Y Zhao, X Su, SC Chen. Genetic alterations activating kinase and cytokine receptor signaling in high-risk acute lymphoblastic leukemia. . Cancer cell 22 (2), 153-166  
PA Northcott, DJH Shih, J Peacock, L Garzia, AS Morrissy, T Zichner. Subgroup-specific structural variation across 1,000 medulloblastoma genomes. . Nature 488 (7409), 49-56  
Cancer Genome Atlas Network. Comprehensive molecular characterization of human colon and rectal cancer. . Nature 487 (7407), 330-337  
SP Shah, A Roth, R Goya, A Oloumi, G Ha, Y Zhao, G Turashvili, J Ding. The clonal and mutational evolution spectrum of primary triple-negative breast cancers. . Nature 486 (7403), 395-399  
A Hadj Khodabakhshi, RD Morin, AP Fejes, AJ Mungall, KL Mungall. Recurrent targets of aberrant somatic hypermutation in lymphoma. . Oncotarget 3 (11)  
D Yorukoglu, F Hach, L Swanson, CC Collins, **I Birol**, SC Sahinalp. Dissect: detection and characterization of novel structural alterations in transcribed sequences. . Bioinformatics 28 (12), i179-i187  
M Sze, PA Dimitriu, JV Gosselink, WM Elliott, MM Moore, S Adams. A Preliminary Assessment Of Fungi In The Lung Microbiome. . C52. COPD PATHOGENESIS: HUMAN STUDIES, A4537-A4537  
